common/models/zod-schemas.ts

```diff
@@ -41,3 +41,17 @@ export const OttApiRequestAddToQueueSchema = z.union([
 export const OttApiRequestRemoveFromQueueSchema = z.object({
 	...VideoIdSchema.shape,
 });
+
+export const OttApiRequestAccountRecoveryStartSchema = z.union([
+	z.object({
+		email: z.string().email(),
```

Comment:
The test could be failing because technically we didn't validate this field as an email before. We probably should though. Its possible i used an email that zod thinks is invalid in the test.